src/Microsoft.ML.FastTree/GamModelParameters.cs

```diff
@@ -322,6 +335,19 @@ public double[] GetFeatureWeights(int featureIndex)
             return featureWeights;
         }
 
+        /// <summary>
+        /// Get all the binned weights.
+        /// </summary>
+        public double[][] GetBinEffects()
+        {
+            double[][] featureWeights = new double[_binEffects.Length][];
+            for (int i = 0; i < featureWeights.Length; i++)
+            {
+                featureWeights[i] = GetBinEffects(i);
```

Comment:
>featureWeights [](start = 16, length = 14)

Would you be much better off by doing the _binEffects.CopyTo instead of calling the overload?

Reply:
Done.